.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: "Publish Cibyl to PyPI"
+on:
+  push:
+    branches:
+      - main
+    tags:
+      # asterisk allow matching tags like v1.0.0.rc1, etc.
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.9']
```

Comment:
We have been targeting python 3.6 this whole time. Let's use that version instead, it will make cibyl compatible with RHEL 8.

Reply:
In theory it should not matter, since we are building source distributions and wheels, but we can set to 3.6 to make sure.

Reply:
Being such a small change, let's have it just to be more consistent.